app/database/models_v2.py

```diff
@@ -0,0 +1,310 @@
+"""SQLAlchemy database models."""
```

Comment:
I think `models.py` is a good name. If there's already `models.py` file, you can `git move` it to `models_old.py`